CryptoPrices/Sources/Home/Sources/Home/CommonViews/Header.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Header.swift
+//  Home
+//
+//  Created by Doan Thieu on 05/01/2023.
+//
+
+import Styleguide
+import SwiftUI
+
+struct Header: View {
```

Comment:
Do you think we should be explicit about the Header name here that we should turn into `HomeHeader` instead? As this header layout looks kinda specific for the Home screen and we might have different type of headers in the future. What do you think? 🤔

Reply:
Yes, I totally agree 👍 I'll make it updated!